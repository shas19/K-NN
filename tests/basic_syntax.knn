let a = 3 in 
let b = 5 in 
let c = 7 in 
a*(b+c)