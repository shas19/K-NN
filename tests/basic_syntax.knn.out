<T>
  <k>
    0.36000000000000000e2 ~> .
  </k>
  <env>
    .Map
  </env>
  <store>
    .Map
  </store>
  <nextLoc>
    0
  </nextLoc>
  <shape>
    .Map
  </shape>
</T>
